src/apps.json

```
@@ -8868,7 +8868,7 @@
       "script": [
         "react(?:-with-addons)?[.-]([\\d.]*\\d)[^/]*\\.js\\;version:\\1",
         "/([\\d.]+)/react(?:\\.min)?\\.js\\;version:\\1",
-        "react.*\\.js"
+        "^react\\.(?:production|development)?\\.*(?:min\\.)?js"
```

Would a simple word boundary do the trick here?

```suggestion
        "react\\b.*\\.js"
```

---

Yep that should work :)